apple-nav/src/comp/navwrapper/NavWrapper.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Since this is just rendering TopNav I honestly don't know what would stop you from condensing this.
I'm sure I'm missing something, shoot me a message if I'm missing something.